requirements/test.txt

```diff
@@ -14,3 +14,4 @@ fire
 
 cloudpickle>=1.3
 scikit-learn>1.0, <1.1.1
+pytorch-msssim==0.2.1
```

Comment:
update this file instead:
https://github.com/Lightning-AI/metrics/blob/master/requirements/image_test.txt
```suggestion
```